### PR TITLE
Cleanup and nest commands

### PR DESCRIPTION
**Pull request by KenwoodFox**
Move the instant and "simple" commands into a folder called `functions`

### CODE REVIEW

**Comment by KenwoodFox** [comment]
Why is nobody reviewing these @boinary @KhanSimeoni @cocothebam

**Comment by GregIngersoll** [comment]
Hi @KenwoodFox -- Greg (Mentor) here.
This is my experience from working with git in a team environment (5-10 people) -- others may have different experiences.
For my PRs at Autodesk, nobody will review a PR unless they are tagged as a reviewer.
Also, I usually have to poke people at least once in Slack to review.

I think three members of the coding team are on Discord, so notifying people there might help.

This was actually the first PR that Erika (erikanook1@gmail.com) was notified about, so she didn't know about this before yesterday.

I hope this answers your question.

Erika and I will both attend the meeting this afternoon, at which time it would be nice to discuss a few things.

Thanks!

**Comment by KenwoodFox** [comment]
Sounds great thank you @GregIngersoll Ill see you guys at the meeting

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -16,9 +16,9 @@
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import frc.robot.commands.HumanControl;
 import frc.robot.commands.ROSControl;
-import frc.robot.commands.ResetEncoders;
-import frc.robot.commands.ShiftDown;
-import frc.robot.commands.ShiftUp;
+import frc.robot.commands.functions.ResetEncoders;
+import frc.robot.commands.functions.ShiftDown;
+import frc.robot.commands.functions.ShiftUp;
 import frc.robot.subsystems.Drivetrain;
```
Comment (boinary) [inline src/main/java/frc/robot/RobotContainer.java]: The purpose of this PR is to unnest some commands.

Looks like you did just that! I approve

Comment (KenwoodFox) [reply src/main/java/frc/robot/RobotContainer.java]: Huzzah!

**Pull request merged**